test/egithub_webhook_req_tests.erl

```diff
@@ -0,0 +1,68 @@
+%%%-------------------------------------------------------------------
+%%% @author mwahl
+%%% @copyright (C) 2016, <COMPANY>
+%%% @doc
+%%%
+%%% @end
+%%% Created : 28. Oct 2016 11:40 AM
+%%%-------------------------------------------------------------------
```

Comment:
Please remove or fix these comments (from line 1 to 8). I'd let only de @doc section, so you can add a brief description about the module.